src/std/net/address.ss

```diff
@@ -14,7 +14,26 @@
         ip6-address-string? ip6-address->string string->ip6-address
         inet-address? inet-address
         inet-address-string? inet-address->string string->inet-address
-        resolve-address resolved-address?)
+        resolve-address resolved-address?
+        inaddr-any4 inaddr-any6
+        localhost4 localhost6)
+
+(def inaddr-any4
+  (make-u8vector 4 0))
+
+(def inaddr-any6
+  (make-u8vector 16 0))
+
+(def localhost4
```

Comment:
What about the /8?

Reply:
when we have cidr subnets, some time in v19.